test/e2e_node/services/kubelet.go

```diff
@@ -190,6 +197,58 @@ func createPodManifestDirectory() (string, error) {
 	return path, nil
 }
 
+// createKubeconfig creates a kubeconfig file at the fully qualified `path`.
+func createKubeconfig(path string) error {
+	kubeconfig := `apiVersion: v1
+kind: Config
+users:
+- name: kubelet
+clusters:
+- cluster:
+    server: ` + getAPIServerClientURL() + `
+    insecure-skip-tls-verify: true
```

Comment:
we don't want real deployments running with unsecured localhost, so we want to run our tests with the fully secured environment to flush out assumptions that unsecured localhost is available